hub/api/tests/test_dataset.py

```diff
@@ -868,6 +868,29 @@ def test_dataset_name():
     assert ds3.name == "my_dataset_2"
 
 
+def test_endpoint():
+    """Access credentials shown in this example belong to https://play.min.io:9000.
+    These credentials are open to public. Feel free to use this service for testing and development. Replace with your own MinIO keys in deployment."""
+
+    token = {
+        "aws_access_key_id": "Q3AM3UQ867SPQQA43P2F",
```

Comment:
Same here, please create new credentials and store them in CirceCI for passing tests.